devices/wholeBodyDynamics/WholeBodyDynamicsDevice.h

```diff
@@ -140,6 +144,7 @@ class wholeBodyDynamicsDeviceFilters
  * |                |   portName_1   | string (name of the port opened to stream the external wrench | - | - | Yes    | Bottle of three elements describing the wrench published on the port: the first element is the link of which the published external wrench is applied. This wrench is expressed around the origin of the frame named as second paramter, and with the orientation of the third parameter.  |  |
  * |                |   ...   | | - | ..                                        | Yes       | ..  |  |
  * |                |   portName_n   | .. | - | -                               | Yes       | ..  | |
+ * | outputWrenchPortInfoType |   -   | string | - | netWrench | No       | Variable for choosing the type of information published in the YARP ports of the group "WBD_OUTPUT_EXTERNAL_WRENCH_PORTS". The user can select between "netWrench" and "contactWrenches"  | If an invalid value is put, the device ignores it and "netWrench" is selected |
```

Comment:
Can you briefly explain what `netWrench` and `contactWrenches` do?

Reply:
Done in https://github.com/robotology/whole-body-estimators/pull/48/commits/0f76c5b94fe976b3afc7b5dfd0e3a8cb3adfb022